opengever/ogds/models/query.py

```diff
@@ -11,3 +14,28 @@ def _attribute(self, name):
 
         """
         return _entity_descriptor(self._joinpoint_zero(), name)
+
+    def by_searchable_text(self, text=''):
```

Comment:
hm i wonder - maybe we should move input sanitation to the controller and let this method accept a list of search terms. what do you think?